### PR TITLE
JFFI: Consolidate configuration into tox.ini.

### PR DESCRIPTION
**Pull request by jason-curtis**
pytest and flake8 already look there, so why not?

Plus there is an issue with atom in particular where for some reason, flake8 does not look in setup.cfg for configuration.

### CODE REVIEW

**Review by EvanSimpson — APPROVED** [review]

**Review by jaimemarijke — APPROVED** [review]
👍 to fewer config files.

File: tox.ini [hunk 1]
```diff
@@ -11,3 +11,11 @@ deps =
     -rtest_requirements.txt
 commands =
     pytest []
+
+[flake8]
+exclude = .git,__pycache__,.eggs,.pytest_cache,build,dist,lib
+max_line_length = 120
+ignore = W503,E203
```
Comment (rush340) [inline tox.ini:18]: why E203?

Comment (jaimemarijke) [reply tox.ini:18]: good question. I don't remember any particular conversation about it. Looking at the rule, I would prefer to have the rule enforced, not ignored. https://lintlyci.github.io/Flake8Rules/rules/E203.html

Comment (jason-curtis) [reply tox.ini:18]: because it's not pep8 and black has reason to violate it sometimes.

https://github.com/python/black#slices

**Review by rush340 — APPROVED** [review]

**Pull request merged**